From 7fc2d41e9ffc8f8ed9a42dad3e3f84c8ed827b08 Mon Sep 17 00:00:00 2001
From: svenate <salini.venate@intel.com>
Date: Tue, 15 Mar 2022 23:32:40 +0530
Subject: [PATCH] Remove -fno-strict-overflow to fix Android T build errors

This is a WA patch to fix below build error in
Android T:
clang: error: argument unused during compilation:
 '-fno-strict-overflow' [-Werror,-Wunused-command-line-argument]

Signed-off-by: svenate <salini.venate@intel.com>
---
 hdmi/Android.mk    | 1 -
 primary/Android.mk | 1 -
 usb/Android.mk     | 1 -
 3 files changed, 3 deletions(-)

diff --git a/hdmi/Android.mk b/hdmi/Android.mk
index a7b237c..2144962 100755
--- a/hdmi/Android.mk
+++ b/hdmi/Android.mk
@@ -36,7 +36,6 @@ LOCAL_C_INCLUDES += \
     external/tinyalsa/include
 
 LOCAL_CFLAGS :=\
- -fno-strict-overflow \
  -fwrapv \
  -D_FORTIFY_SOURCE=2 \
  -fstack-protector-strong \
diff --git a/primary/Android.mk b/primary/Android.mk
index c127ba0..d60e6a7 100755
--- a/primary/Android.mk
+++ b/primary/Android.mk
@@ -37,7 +37,6 @@ LOCAL_C_INCLUDES += \
 	$(call include-path-for, audio-effects)
 
 LOCAL_CFLAGS :=\
- -fno-strict-overflow \
  -fwrapv \
  -D_FORTIFY_SOURCE=2 \
  -fstack-protector-strong \
diff --git a/usb/Android.mk b/usb/Android.mk
index 25fd107..f4bd3e6 100755
--- a/usb/Android.mk
+++ b/usb/Android.mk
@@ -37,7 +37,6 @@ LOCAL_C_INCLUDES += \
 	$(call include-path-for, audio-effects)
 
 LOCAL_CFLAGS :=\
- -fno-strict-overflow \
  -fwrapv \
  -D_FORTIFY_SOURCE=2 \
  -fstack-protector-strong \
-- 
2.17.1

